crates/sui-storage/src/event_store/sql.rs

```diff
@@ -329,34 +326,52 @@ impl From<SqliteRow> for StoredEvent {
     }
 }
 
-const TS_QUERY: &str =
-    "SELECT * FROM events WHERE timestamp >= ? AND timestamp < ? ORDER BY timestamp ASC LIMIT ?";
+const ALL_QUERY_ASC: &str = "SELECT * FROM events WHERE seq_num >= ? ORDER BY seq_num ASC LIMIT ?";
+const ALL_QUERY_DESC: &str =
+    "SELECT * FROM events WHERE seq_num <= ? ORDER BY seq_num DESC LIMIT ?";
```

Comment:
Hmm, I wonder if there is a way to avoid all this repetition of the query except for the ASC/DESC part.  Like, we define a single query, but the ASC/DESC can be replaced by something like SEQ_ORDER.  Then a function can replace SEQ_ORDER with ASC/DESC.  This would help prevent accidental mistakes - right now we have no way of ensuring the two queries are the same.

Reply:
There are no easy way using the SQL's api, I will write a query builder for this to prevent accidental mistakes